src/Spice86.Core/Emulator/InterruptHandlers/Dos/DosInt21Handler.cs

```diff
@@ -67,6 +67,10 @@ public DosInt21Handler(IMemory memory, Cpu cpu,
         FillDispatchTable();
     }
 
+    public override void Run(int index) {
+        base.Run(index);
+    }
+
```

Comment:
What does this do?

Reply:
No changes compared to master. This is a false positive.